SECURITY.md

```diff
@@ -0,0 +1,9 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+If you believe you've found something in Django REST Framework JSON API which has security implications, please **do not raise the issue in a public forum**.
+
+Send a description of the issue via email to [whatemailaddressshouldweuse@example.net][security-mail].The project maintainers will then work with you to resolve any issues where required, prior to any public disclosure.
```

Comment:
A good question. Maybe `rest-framework-jsonapi-security@googlegroups.com` where the members of that google group are the maintainer or core team? Someone will have to remember to update the group membership whenever maintainers change -- which is not often.

Reply:
Created. I will update the PR later on. But could you share the email address which I should add to the security mailing list?

Reply:
@sliverc See maintainers discussion